src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/AwakePage.xaml

```diff
@@ -29,86 +29,50 @@
-                                                   LineHeight="20">
-                                            <Run x:Uid="Awake_NoKeepAwakeContent"/>
-                                            <LineBreak/>
-                                            <Run Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                x:Uid="Awake_NoKeepAwakeDescription"/>
-                                        </TextBlock>
-                                    </RadioButton.Content>
-                                </RadioButton>
-                                <RadioButton x:Uid="Awake_IndefiniteKeepAwake"
-                                             IsChecked="{x:Bind Path=ViewModel.Mode, Mode=TwoWay, Converter={StaticResource AwakeModeToBoolConverter}, ConverterParameter=1}">
-                                    <RadioButton.Content>
-                                        <TextBlock TextWrapping="WrapWholeWords"
-                                                   LineHeight="20">
-                                            <Run x:Uid="Awake_IndefiniteKeepAwakeContent"/>
-                                            <LineBreak/>
-                                            <Run Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                 x:Uid="Awake_IndefiniteKeepAwakeDescription"/>
-                                        </TextBlock>
-                                    </RadioButton.Content>
-                                </RadioButton>
-                                <RadioButton x:Uid="Awake_TemporaryKeepAwake"
-                                            IsChecked="{x:Bind Path=ViewModel.Mode, Mode=TwoWay, Converter={StaticResource AwakeModeToBoolConverter}, ConverterParameter=2}">
-                                    <RadioButton.Content>
-                                        <TextBlock TextWrapping="WrapWholeWords"
-                                                   LineHeight="20">
-                                            <Run x:Uid="Awake_TemporaryKeepAwakeContent"/>
-                                            <LineBreak/>
-                                            <Run Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                 x:Uid="Awake_TemporaryKeepAwakeDescription"/>
-                                        </TextBlock>
-                                    </RadioButton.Content>
-                                </RadioButton>
-                                <StackPanel AutomationProperties.LabeledBy="{Binding ElementName=ModeTitleLabel}"
-                                            Margin="0,-8,0,0">
-
-                                    <StackPanel Margin="28,8,0,0"
+                    <controls:Setting x:Uid="Awake_TimeBeforeAwake" Icon="&#xE121;" Visibility="{x:Bind ViewModel.IsTimeConfigurationEnabled, Mode=OneWay}">
```

Comment:
#10331 - use `e823` "Recent" for the same icon or `e916` "Stopwatch". Ideally the same as "timer" in: 
![image](https://user-images.githubusercontent.com/65828559/135680118-e5da4c2a-e60e-4ec2-ab8b-ee044246a9b0.png)

Reply:
Thanks, replaced it with the stopwatch icon